model/pyproject.toml

```diff
@@ -42,7 +42,7 @@ tests = [
 ]
 
 [tool.setuptools]
-py-modules = ["model_training"]
+packages = ["model_training"]
```

Comment:
I am unsure why this change is shown, this was already merged in https://github.com/LAION-AI/Open-Assistant/pull/3643.